doc/Security.xml

```diff
@@ -1035,6 +1035,14 @@
         <para>The ValueType attribute shall have the value of</para>
         <programlisting>urn:ietf:params:oauth:token-type:jwt</programlisting>
       </section>
+      <section>
+        <title>Usage of JWT-based client authentication over RTSP</title>
+        <para>Since authentication tokens contain sensitive information that could be easily used to perform replay attacks, JWT-based client authentication should not be used over unencrypted RTSP connections.</para>
```

Comment:
```suggestion
        <para>Since authentication tokens contain sensitive information that could be easily used to perform replay attacks, JWT-based client authentication must not be used over unencrypted RTSP connections.</para>
```

Reply:
changed to shall